ow_plexil/src/plexil-adapter/TerminalPlanSelection.h

```diff
@@ -0,0 +1,31 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef Terminal_Plan_Selection_H
+#define Terminal_Plan_Selection_H
+
+#include <ros/ros.h>
+#include <ow_plexil/PlanSelectionCommand.h>
+#include <std_msgs/String.h>
+
+class TerminalPlanSelection{
+  public:
+    TerminalPlanSelection() = default;
+    ~TerminalPlanSelection() = default;
+    TerminalPlanSelection(const TerminalPlanSelection&) = delete;
+    TerminalPlanSelection& operator = (const TerminalPlanSelection&) = delete;
+    void planSelectionStatusCallback(const std_msgs::String::ConstPtr&);
+    void initialize();
+    void start(bool);
```

Comment:
I know argument names are optional but they give a hint of what the argument is about:
```suggestion
    void start(bool initial_plan);
```

Reply:
also applies to other places in your code.